src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -84,157 +87,175 @@
-                             [:div {}
-                              [:label {:style {:cursor "pointer"}}
-                               [:input {:type "checkbox"
-                                        :checked (not (false? stateval))
-                                        :onChange #(swap! state update-in [:selected-types label] false?)
-                                        :style {:cursor "pointer"}}]
-                               [:span {:style {:marginLeft "0.5ex"}} label]]]))]
-            [overlay/Overlay {:get-anchor-dom-node #(react/find-dom-node (@refs "anchor"))
-                              :anchor-x :right :anchor-y :bottom
-                              :dismiss-self #(swap! state dissoc :show-access-level-select?)
-                              :content
-                              (react/create-element
-                               [:div {:style {:padding "1em" :border style/standard-line}}
-                                (map checkbox access-types)
-                                [:hr {:style {:size "1px" :noshade true}}]
-                                (map checkbox dataset-types)
-                                [:hr {:style {:size "1px" :noshade true}}]
-                                (map checkbox realm-types)])}]))
-        [table/Table
-         {:state-key "workspace-table"
-          :empty-message "No workspaces to display." :retain-header-on-empty? true
-          :cell-padding-left nil
-          :initial-rows-per-page 10
-          :header-row-style {:fontWeight nil :fontSize "90%"
-                             :color (:text-lighter style/colors) :backgroundColor nil}
-          :header-style {:padding "0.5em 0 0.5em 14px"}
-          :resizable-columns? true :reorderable-columns? false :resize-tab-color (:line-default style/colors)
-          :body-style {:fontSize nil :fontWeight nil
-                       :borderLeft style/standard-line :borderRight style/standard-line
-                       :borderBottom style/standard-line}
-          :row-style {:height row-height-px :borderTop style/standard-line}
-          :cell-content-style {:padding nil}
-          :toolbar (table-utils/add-right
-                    [create/Button (select-keys props [:nav-context :billing-projects :disabled-reason])])
-          :filter-groups [{:text "All" :pred (constantly true)}
-                          {:text "Complete" :pred #(= "Complete" (:status %))}
-                          {:text "Running" :pred #(= "Running" (:status %))}
-                          {:text "Exception" :pred #(= "Exception" (:status %))}]
+   (fn [{:keys [props state this]}]
+     (let [{:keys [workspaces nav-context]} props
+           {:keys [filters-expanded?]} @state
+           max-workspace-name-length (get-max-length get-workspace-name-string workspaces)
+           max-description-length (get-max-length get-workspace-description workspaces)]
+       [Table
+        {:ref "table" :persistence-key "workspace-table" :v 2
```

Comment:
There are two versions here: the page, and the table (and I just noticed that the page persistence key is still "workspace-table-types" -- do we want to do something about that?).  We could tie them together for clarity though.